pkg/constants/constants.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+const (
```

Comment:
Indices names are build from an alias + counter. The alias (e.g. `app`) is constant, while a counter can be increased. In this case, we'll miss logs in the `app-000002` index, for example.